erc20/gno/README.md

```diff
@@ -0,0 +1,59 @@
+# Formal Verification of GNO ERC20 Token
+
+By Dominik Teiml (dominik@gnosis.pm), [Gnosis](https://www.gnosis.pm)
+
+We formally verified the GNO ERC20 token contract [runtime bytecode](./gno-erc20.bytes).
+
+We found the following deviations from [ERC20-EVM](../vyper/vyper-erc20-spec.ini):
+
+- In the [high-level Solidity code](https://etherscan.io/address/0x6810e776880c02933d47db1b9fc05908e5386b96#code), mapping of users' allowances is called `allowed`, hence we use that in our [eDSL spec](./gno-erc20-spec.ini).
+- In `transfer-failure-1` and `transferFrom-failure-1`, `BAL_TO +Int VALUE >=Int (2 ^Int 256)` was removed from the constraint. The reason is the GNO was constructed with a fixed supply of 10 M. Hence no overflow protections are necessary (and so none are present)
```

Comment:
I'm not sure about this reasoning, unless you additionally prove that the sum of all balances is equal to the total supply.

Reply:
I see.  I'll think about the best way for that tomorrow